### PR TITLE
Write simple mongod tests

### PR DESCRIPTION
**Pull request by Kimchelly**
* Runs multiple mongods concurrently
* Currently only tests blocking processes

### CODE REVIEW

File: process_test.go [hunk 1]
```diff
@@ -166,3 +173,171 @@ func TestProcessImplementations(t *testing.T) {
...
+
+func teardownMongods(dbPaths []string) {
+	for _, dbPath := range dbPaths {
+		os.RemoveAll(dbPath)
+	}
+}
+
+func TestMongod(t *testing.T) {
+	if testing.Short() {
+		fmt.Println("SHORT")
```
Comment (tychoish) [inline process_test.go]: remove this line

File: process_test.go [hunk 2]
```diff
@@ -166,3 +173,171 @@ func TestProcessImplementations(t *testing.T) {
...
+		dbPaths[i] = dbPath
+
+		opts := CreateOptions{Args: []string{mongodPath, "--port", fmt.Sprintf("%d", port), "--dbpath", dbPath}}
+		optslist[i] = opts
+	}
+
+	return optslist, dbPaths, nil
+}
+
+func teardownMongods(dbPaths []string) {
```
Comment (tychoish) [inline process_test.go]: this function should probably be "removeDBPaths"

File: process_test.go [hunk 3]
```diff
@@ -166,3 +173,171 @@ func TestProcessImplementations(t *testing.T) {
 		})
 	}
 }
+
+// Returns path to release and to mongod
+func downloadMongodb(t *testing.T) (string, string) {
```
Comment (tychoish) [inline process_test.go]: maybe move these tests and their helpers to a different file. totally optional

File: process_test.go (hunk 1 above)
Comment (Kimchelly) [reply process_test.go]: Done

File: process_test.go (hunk 2 above)
Comment (Kimchelly) [reply process_test.go]: Done

File: process_test.go (hunk 3 above)
Comment (Kimchelly) [reply process_test.go]: Done

**Pull request merged**